src/lib/components/no-result/terra-no-result-notice.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+import {
+    ComponentFixture,
+    TestBed
+} from '@angular/core/testing';
+import { TerraNoResultNoticeComponent } from './terra-no-result-notice.component';
+import {
+    TerraButtonComponent,
+    TooltipDirective
+} from '../..';
+import { By } from '@angular/platform-browser';
+import { noResultsNoticeMockButtons } from '../../testing/mock-buttons';
+import {
+    DebugElement,
+} from '@angular/core';
+import { Router } from '@angular/router';
+import { MockRouter } from '../../testing/mock-router';
+
+fdescribe('TerraNorResultNoticeComponent', () =>
+{
+    let fixture:ComponentFixture<TerraNoResultNoticeComponent>;
+    let component:TerraNoResultNoticeComponent;
+    const router:MockRouter = new MockRouter();
+
+    beforeEach(() =>
+    {
+        fixture = TestBed.configureTestingModule({
+                declarations: [TooltipDirective,
+                               TerraButtonComponent,
+                               TerraNoResultNoticeComponent],
+                providers:    [
+                    {
+                        provide:  Router,
+                        useValue: router
+                    }]
```

Comment:
Wenn du den TerraButton mockst, dann wäre das setup etwas simpler.
```suggestion
                declarations: [
                     TerraButtonMockComponent
                     TerraNoResultNoticeComponent
                ]
```
Der Mock ist auch recht simpel:
```
@Component({ selector: 'terra-button', template: '' })
class TerraButtonMockComponent extends TerraButtonComponent {}
```

Reply:
danke :)